### PR TITLE
Adding a couple missing highcharts properites

### PR DESCRIPTION
**Pull request by rustedgrail**
We are just moving over to TS, so there will likely be more PRs or additions to this one as we continue the transition. I re-ordered the axis options and axis options array to provide a better type error. As originally written, the error was about the value not having length (because it wasn't an array) instead of complaining about the properties of the axis options object.

case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

### CODE REVIEW

File: highcharts/highcharts.d.ts [hunk 1]
```diff
@@ -4867,6 +4878,11 @@ interface HighchartsIndividualSeriesOptions {
      * The name of the series as shown in the legend, tooltip etc.
      */
     name?: string;
+ 	/**
+ 	 * A pixel value specifying a fixed width for each column or bar. When null, the width is calculated from
+	 * the pointPadding and groupPadding.
+ 	 */
+ 	pointWidth?: number;
```
Comment (yuit) [inline highcharts/highcharts.d.ts]: The indentation seems off here

File: highcharts/highcharts.d.ts [hunk 2]
```diff
@@ -6098,6 +6114,7 @@ interface HighchartsPointObject {
      */
     category: string | number;
     name: string;
+	index: number;
```
Comment (yuit) [inline highcharts/highcharts.d.ts]: The indentation seems off

**Comment by yuit** [comment]
@rustedgrail thanks for the PR. could you fix the indentation? look good otherswise

**Comment by rustedgrail** [comment]
Fixed the indentation.

File: highcharts/highcharts.d.ts (hunk 2 above)
Comment (yuit) [inline highcharts/highcharts.d.ts]: nit: could you fix the indentation here as well?

**Comment by rustedgrail** [comment]
Do you want me to squash the commits?

**Comment by yuit** [comment]
@rustedgrail  it is cool. I will squash when merge. Thanks for the PR!

**Pull request merged**